ElfUtils/ElfFile.cpp

```diff
@@ -50,6 +50,7 @@ class ElfFileImpl : public ElfFile {
   std::string build_id_;
   bool has_symtab_section_;
   bool has_debug_info_section_;
+  llvm::ArrayRef<uint8_t> text_section_;
```

Comment:
Doesn't this increase memory consumption? Maybe load it only when it is needed?

Reply:
Sorry, I played around with it and this was not supposed to be committed. I am going to remove it.